src/query/api/v1/handler/json/write.go

```diff
@@ -124,7 +124,9 @@ func (h *WriteJSONHandler) parseRequest(r *http.Request) (*WriteQuery, *xhttp.Pa
 	}
 
 	var writeQuery *WriteQuery
-	json.Unmarshal(js, &writeQuery)
+	if err = json.Unmarshal(js, &writeQuery); err != nil {
```

Comment:
Nit: I would do `err :=` here; that way we're definitely only assigning within that scope.

Reply:
The err doesn't persist past this scope, don't know if that changes your mind at all 📦

Reply:
Yeah something of a style thing; we don't need beyond the `if` scope, so might as well leave it in the smaller scope. Not hugely important.

Reply:
Sure, linter sometimes complains about shadowing; doesn't make a difference logically at least :)